### PR TITLE
MRG press 'h' to hide or show all in report

### PR DESCRIPTION
**Pull request by mainakjas**
closes https://github.com/mne-tools/mne-python/issues/1996

### CODE REVIEW

**Comment by mainakjas** [comment]
@teonlamont does this work for you?

File: doc/source/whats_new.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ Changelog
 
     - Add support for scaling and adjusting the number of channels/time per view by `Jaakko Leppakangas`_
 
+    - Add support to hide/show all sections with a single keypress ('h') in ``mne.Report`` by `Mainak Jas`_
```
Comment (Eric89GXL) [inline doc/source/whats_new.rst]: @mainakjas try to get the new syntax going, I think you can do `:class:` in front of a single-ticked `mne.Report` to get it to work

File: mne/report.py [hunk 2]
```diff
@@ -716,6 +728,10 @@ class Report(object):
         timepoints t such that a <= t <= b.
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+
+    Notes
+    -----
+    To hide/show all sections in the html report, press 'h'
```
Comment (Eric89GXL) [inline mne/report.py]: Can you see if it's possible to get the `.. versionadded` sensibly part of this statement? Does it render okay in a parenthetical?

**Comment by mainakjas** [comment]
@Eric89GXL comments addressed

**Comment by agramfort** [comment]
works like charm ! thanks @mainakjas  !

**Comment by teonlamont** [comment]
awesome!!!! thanks @mainakjas, haven't tried it yet but I like the idea

**Comment by mainakjas** [comment]
... woops, I was going to fix something. I should have changed the status of the PR. My bad. It's actually not showing/hiding all but rather toggling every button. I can make another mini PR to change the docs & make the keyboard shortcut 't'. Would that work?

**Comment by teonlamont** [comment]
wait, what's going on? it (un)checks the buttons but the plots are still shown?

**Comment by agramfort** [comment]
for me it hides everything....

I ran it from the command line

**Comment by mainakjas** [comment]
ok looks like we need another PR definitely. @teonlamont does it work for you if you run from the command line? Did you try other browsers?

**Comment by teonlamont** [comment]
ah, so at a first glance, it does work beautifully. @mainakjas just mentioned and I can also concur that if you then click one, then press 'h', that's where things get interesting: instead of hiding the click plot, it toggles all the others on. 
I think default for 'h' should be if any plots are selected, and you press 'h', it hides them all, then if you press it again, all the plots appear.

**Comment by teonlamont** [comment]
and if you wanted a toggle mode, 't', if you press it then click a plot, it would only highlight the one you click instead of add.

**Comment by agramfort** [comment]
@teonlamont please keep things simple. Using h for hide and t for toggle is
simple enough. Maybe s could be used to show all. @mainakjas showed you how
to do it.

**Comment by mainakjas** [comment]
cool, I fixed it.

**Comment by teonlamont** [comment]
yeah, I agree it should be simple. I was just trying to spelt all the fringe cases.

**Pull request merged**